app/Models/product_category.php

```diff
@@ -5,7 +5,7 @@
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
 
-class product_category extends Model
+class Product_Category extends Model
```

Comment:
Class names are UpperCamelCase
```suggestion
class ProductCategory extends Model
```